recipe/meta.yaml

```
@@ -48,7 +48,6 @@ outputs:
     # it is here to keep-up with the legacy name, containing an
     # underscore ("_") instead of a dash.
     build:
```

Regardless of if this would help with the linter error or not, this should be `noarch: generic`.
The reason for that is that `noarch: python` combines two properties into one (due to historical reasons):
1. Make the package available as `noarch` and not in a platform-dependent subdir.
2. Do some special handling while packaging/installing the package's content only relevant for Python packages. (This is most importantly to make a single build of a package installable for different Python versions, i.e., install the packages into `lib/pythonX.Y/...` for any Python version `X.Y`.)

If we have an otherwise empty meta-package, we only need point 1 and not point 2, so it could/should be `noarch: generic`. `noarch: python` should only be used for the actual Python package it depends on.
```suggestion
    build:
      noarch: generic
```